src/app/entity-metadata.ts

```diff
@@ -7,17 +7,17 @@ export const entityMetadata: EntityMetadataMap = new Map([
       entityType: ObservabilityEntityType.Api,
       icon: ObservabilityIconType.Api,
       detailPath: (id: string, sourceRoute?: string) => [sourceRoute ?? '', 'endpoint', id],
-      sourceRoutes: ['apis']
+      sourceRoutes: ['endpoints']
     }
   ],
   [
     ObservabilityEntityType.Service,
     {
       entityType: ObservabilityEntityType.Service,
       icon: ObservabilityIconType.Service,
-      detailPath: (id: string) => ['apis', 'service', id],
-      listPath: ['apis'],
-      apisListPath: (id: string) => ['apis', 'service', id, 'endpoints'],
+      listPath: ['endpoints'],
```

Comment:
What are the desired breadcrumbs? Urls? I think things are a little off here (in terms of the entity metadata, the urls and the breadcrumb resolvers). 

I think I'd expect:
Endpoint List => '/endpoints'
Endpoint => '/endpoint/<id>' (should it be a different url if reached from service endpoint list vs endpoint list? different breadcrumb? If yes, then also '/service/endpoint/<id>' and matching source path)
Service => '/service/<id>' ; no list path, updated detail + api list path
Service Endpoint List => '/service/<id>/endpoints

Reply:
I _think_ I agree. Here is what we will have after the incoming changes.

<img width="642" alt="Screen Shot 2021-01-21 at 1 19 28 PM" src="https://user-images.githubusercontent.com/45181984/105414344-371d2b00-5bec-11eb-984d-c932aabd2f3a.png">
<img width="656" alt="Screen Shot 2021-01-21 at 1 19 38 PM" src="https://user-images.githubusercontent.com/45181984/105414368-3be1df00-5bec-11eb-8a97-fcb75d1de15c.png">

<img width="672" alt="Screen Shot 2021-01-21 at 1 19 49 PM" src="https://user-images.githubusercontent.com/45181984/105414381-40a69300-5bec-11eb-85ef-03f7f172fc50.png">

Reply:
I think the one that is still a bit of a question is endpoint.

My feelings are (assuming we want to show the service breadcrumb):
- When navigated from a service: `/service/endpoint/<endpoint-id>`. This would correspond to breadcrumbs of: `<Service Name> > Endpoints > <Endpoint name>`
- When navigated from elsewhere, such as EP list: `/endpoint/<endpoint-id>` This would correspond to breadcrumbs of: `Endpoints > <Endpoint name>`

Reply:
- Are we getting rid of the service listing screen? 
- How are we navigating to a service, when there is no service listing page? Via the service column in API endpoints list?

Reply:
Why did we get rid of service screen? This was major thing in a customer perspective and they are using service dashboards for insights. 

Is there an issue or discussion around this as I came to know about this because of latest UI release I cut yesterday. 

I am happy with moving APIs in first class navigation but that shouldn't result in removal of service nav completely.

Reply:
This conversation has happened in a few spots now, but duplicating here:
So in the weeds of the discussions around the endpoint list, it was overlooked that it should be a sibling of the service list, which will now be flat. We'll get that cleaned up today.

Reply:
I put up https://github.com/hypertrace/hypertrace-ui/pull/521 to fix the most pressing issues. It has significant, but not complete overlap with this PR (Sorry!)